### PR TITLE
New Serverless pattern - CloudWatch Logs subscription for Kinesis Firehose - CDK

### PR DESCRIPTION
**Pull request by pubudusj**
*Issue #, if available:*
N/A
*Description of changes:*
Example added for CloudWatch Logs subscription for Kinesis Firehose to S3 with CDK.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cloudwatch-logs-subscription-firehose-cdk/bin/cloudwatch-logs-subscription-firehose-cdk.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { CloudwatchLogsSubscriptionFirehoseCdkStack } from '../lib/cloudwatch-logs-subscription-firehose-cdk-stack';
```
Comment (boyney123) [inline cloudwatch-logs-subscription-firehose-cdk/bin/cloudwatch-logs-subscription-firehose-cdk.ts:4]: Think we are missing the lib folder here?

**Review by boyney123 — CHANGES_REQUESTED** [review]

**Comment by boyney123** [comment]
Thanks again @pubudusj for the submission, looks like we are missing the lib folder for the CDK pattern, once you add it we can review it. Thanks!

File: cloudwatch-logs-subscription-firehose-cdk/bin/cloudwatch-logs-subscription-firehose-cdk.ts (hunk 1 above)
Comment (pubudusj) [reply cloudwatch-logs-subscription-firehose-cdk/bin/cloudwatch-logs-subscription-firehose-cdk.ts:4]: Sorry about this @boyney123 
Forgot to define it in gitignore. This is fixed now.

Comment (boyney123) [reply cloudwatch-logs-subscription-firehose-cdk/bin/cloudwatch-logs-subscription-firehose-cdk.ts:4]: Thanks!

**Comment by boyney123** [comment]
Thanks again for the submission

**Comment by boyney123** [comment]
Pattern will be live at https://serverlessland.com/patterns/cloudwatch-logs-subscription-firehose-cdk

**Comment by pubudusj** [comment]
Thanks @boyney123 👍

**Pull request merged**